GrandCentralBoard/Widgets/Watch/Source/CalendarNameSource.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  CalendarNameSource.swift
+//  GrandCentralBoard
+//
+//  Created by Michał Laskowski on 12.04.2016.
+//  Copyright © 2016 Oktawian Chojnacki. All rights reserved.
+//
+
+import GrandCentralBoardCore
+
+class CalendarNameSource : Asynchronous {
```

Comment:
At first I was trying to use a protocol instead of class, but had problems with casting to it in switch statement. I will try some other ways to make it right